electron/app/components/CheckboxGrid.tsx

```diff
@@ -82,14 +94,19 @@ const Body = styled.div`
       }
 
       span.name {
-        padding-left: 4px;
+        padding: 0 4px;
         white-space: nowrap;
         overflow-x: hidden;
         text-overflow: ellipsis;
         flex-grow: 1;
         max-width: 100%;
         line-height: 24px;
       }
+      span.count {
+        text-overflow: ellipsis;
+        overflow-x: hidden;
+        white-space: nowrap;r
```

Comment:
Extra `r` here (styled-components seems to ignore it, though)

Reply:
Thanks!